index.html

```diff
@@ -68,7 +88,17 @@ <h1 class="Text__intro"> <strong>Hey there. I’m Solomon </strong></h1>  <br> <
                             <img src="img/angellist.svg" alt="twitter Icon" class="social_media__icons">
                         </a>
                     </li>
+
+                    
                 </ul>
+                <!-- Hug Log for Desktop -->
+
+            <div class="hug__Desktop"> 
+                    <a href="#">
+                        <img src="/img/hug.png" alt="Going to the next" class="hug__Desktop__img">
```

Comment:
Please note that this image does not currently show. I suggest you fix the path to this image. It is always great to have your website functioning as intended to ensure user experience is optimised.
<img width="1672" alt="Screenshot 2022-09-30 at 14 03 06" src="https://user-images.githubusercontent.com/48546992/193256687-28be88e4-29b6-4f8b-bf8f-5cd2ebce932e.png">

Reply:
> Please note that this image does not currently show. I suggest you fix the path to this image. It is always great to have your website functioning as intended to ensure user experience is optimised. <img alt="Screenshot 2022-09-30 at 14 03 06" width="1672" src="https://user-images.githubusercontent.com/48546992/193256687-28be88e4-29b6-4f8b-bf8f-5cd2ebce932e.png">

Hello @Nyame-Wolf Thank you for your review; However this this image is actually loading where you run the code on a live-server {Vs code extension} . but if you run directly as an html file it would will not be visible.